metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_queries.py

```diff
@@ -626,13 +627,32 @@ def _parse_audit_log_row(
 
             columns = set()
             for modified_column in obj["columns"]:
-                columns.add(
-                    self.identifiers.snowflake_identifier(modified_column["columnName"])
-                )
+                column_name = modified_column["columnName"]
+                if not column_name or not column_name.strip():
+                    has_empty_column = True
```

Comment:
I would also add some comment explaining why are we deciding to parse the query ourselves in cases where there are empty column names.